tests/plaq_test.cpp

```diff
@@ -79,6 +79,7 @@ void setGaugeParam(QudaGaugeParam &gauge_param)
   pad_size = MAX(pad_size, t_face_size);
   gauge_param.ga_pad = pad_size;
 #endif
+  gauge_param.ga_pad = 0;
```

Comment:
Should this be inside an `#else/#endif`? Otherwise the `gauge_param.ga_pad = pad_size;` in the multigpu block is pointless.